frontend/packages/dev-console/src/components/buildconfig/BuildConfigForm.tsx

```diff
@@ -43,8 +44,9 @@ const BuildConfigForm: React.FC<FormikProps<BuildConfigFormikValues> & {
   const namespace = watchedBuildConfig?.metadata?.namespace || activeNamespace;
 
   const isStale = watchedBuildConfig?.metadata?.resourceVersion !== values.resourceVersion;
+  const [formReload, setFormReload] = useState<number>(0);
```

Comment:
No need to import useState separately when you have whole `React` namespace available.
```suggestion
  const [formReload, setFormReload] = React.useState<number>(0);
```